src/UserActivity.js

```diff
@@ -49,6 +51,14 @@ class UserActivity {
         window.removeEventListener('wheel', this._onUserActivity.bind(this), true);
     }
 
+    /**
+     * Return true if there has been user activity very recently
+     * (ie. within a few seconds)
+     */
+    userCurrentlyActive() {
+        return this.lastActivityAtTs > (new Date).getTime() - CURRENTLY_ACTIVE_THRESHOLD;
```

Comment:
Whilst `new Date` is valid JS, I don't like it, especially when we're nesting parentheses around it. Can we `s/(new Date)/new Date()/` please?